src/core/Microsoft.PowerToys.Settings.UI/Views/ColorPickerPage.xaml.cs

```diff
@@ -21,5 +21,33 @@ public ColorPickerPage()
             DataContext = ViewModel;
             InitializeComponent();
         }
+
+        /// <summary>
+        /// Event is called when the <see cref="ComboBox"/> is completly loaded, inclusive the ItemSource
+        /// </summary>
+        /// <param name="sender">The sender of this event</param>
+        /// <param name="e">The arguments of this event</param>
+        private void ColorPicker_ComboBox_Loaded(object sender, Windows.UI.Xaml.RoutedEventArgs e)
+        {
+           /**
+            * UWP hack
+            * because UWP load the binded ItemSource of the ComboBox asyncronous,
```

Comment:
`asynchronous`

Reply:
Oh, `binded` should probably be `bound`. But the sentence also needs work in that case. (It's too late for me to try to figure out what you're trying to say here.)